Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,44 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+        StartOver:
+            InteractionManager.PerformInteractions();
+            if (InteractionManager.Prompter("Whould you like to start again?").ToLower() == "y")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896164)